src/Symfony/Bundle/FrameworkBundle/KernelBrowser.php

```diff
@@ -123,7 +123,10 @@ public function loginUser(object $user, string $firewallContext = 'main'): self
         }
 
         $token = new TestBrowserToken($user->getRoles(), $user, $firewallContext);
-        $token->setAuthenticated(true);
+        // @deprecated since Symfony 5.4
+        if (method_exists($token, 'setAuthenticated')) {
+            $token->setAuthenticated(true);
+        }
```

Comment:
The real bugfix of this PR, this adds support for security-core 6.0 in FrameworkBundle.